app/car.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import annotations
+
+
+class Car:
+    def __init__(self, brand: str, fuel_consumption: float) -> None:
+        self.brand = brand
+        self.fuel_consumption = fuel_consumption
+
+    @classmethod
+    def create_car(cls, car_info: dict[str, str | float | int]) -> Car:
+        return cls(
+            brand=car_info["brand"],
+            fuel_consumption=car_info["fuel_consumption"]
```

Comment:
To prevent `KeyError` exceptions, consider using the `get` method with default values when accessing dictionary keys, such as `car_info.get('brand', 'default_brand')` and `car_info.get('fuel_consumption', 0.0)`. This will make the code more robust against missing keys.